src/js/components/FilmItem/index.js

```diff
@@ -14,16 +15,18 @@ const FilmItem = (props:Props) => {
 
   return (
     <li className='film-list__item' onClick={() => { onFilmSelect(id) }}>
-      <div className='film-item'>
-        <div className='film-item__img-wrap'>
-          <img alt='film' className='film-item__img' src={film.poster_path} />
+      <Link className='film-list__link' to={{ pathname: `/film/${id}` }}>
```

Comment:
What do you think about mocking `react-router`s components? Since we don't really need those rendered in snapshots.